x-pack/plugins/ml/public/embeddables/anomaly_explorer/anomaly_explorer_embeddable_factory.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import type { StartServicesAccessor } from 'kibana/public';
+
+import type {
+  EmbeddableFactoryDefinition,
+  IContainer,
+} from '../../../../../../src/plugins/embeddable/public';
+import { HttpService } from '../../application/services/http_service';
+import type { MlPluginStart, MlStartDependencies } from '../../plugin';
+import type { MlDependencies } from '../../application/app';
+import {
+  ANOMALY_EXPLORER_CHARTS_EMBEDDABLE_TYPE,
+  AnomalyExplorerEmbeddableInput,
+  AnomalyExplorerEmbeddableServices,
+} from '..';
+import { AnomalyExplorerChartsService } from '../../application/services/anomaly_explorer_charts_service';
+
+export class AnomalyExplorerEmbeddableFactory
+  implements EmbeddableFactoryDefinition<AnomalyExplorerEmbeddableInput> {
+  public readonly type = ANOMALY_EXPLORER_CHARTS_EMBEDDABLE_TYPE;
+
+  constructor(
+    private getStartServices: StartServicesAccessor<MlStartDependencies, MlPluginStart>
+  ) {}
+
+  public async isEditable() {
+    return true;
+  }
+
+  public getDisplayName() {
+    return i18n.translate('xpack.ml.components.mlAnomalyExplorerEmbeddable.displayName', {
+      defaultMessage: 'ML Anomaly Explorer',
```

Comment:
Related to the above, `ML Anomaly Charts` might be a better label for now.

Reply:
Renamed here [`3c31e37` (#94396)](https://github.com/elastic/kibana/pull/94396/commits/3c31e37dacc795ad9e4b0b71d459616bda58b979)